README.md

```diff
@@ -411,12 +411,26 @@ seeds:
       - skip$
 ```
 
+It's possible to do more complex regular expressions. It will only target characters and numbers after search until the word ID= shows, and any numbers after the word ID will be matched. This means that https://example.com/search/6vH8R4TmID=5819 won't be crawled, but https://example.com/search/ID=5819, https://example.com/search/6vH8R4Tm and https://example.com/search/2o3Jq89cID=5ag8h19 will be crawled
+
+```
+seeds:
+  - url: https://the.example.com/startpage.html
+    scopeType: "host"
+    exclude:
+      - *.example.com/search/[A-Za-z0-9]+ID=[0-9]+
+```
+
 The `include`, `exclude`, `scopeType` and `depth` settings can be configured per seed, or globally, for the entire crawl.
 
 The per-seed settings override the per-crawl settings, if any.
 
 The test suite [tests/scopes.test.js](tests/scopes.test.js) for additional examples of configuring scope inclusion and exclusion rules.
 
+It's important to note that the include and exclude rules will always have regular expressions in effect. This means that for your rules to match, you may have to escape special characters that commonly appear in urls like "?", "+", or ".". You just have to place a \ before the character. For example youtube.com/watch\?rdwz7QiG0lk
```

Comment:
Can probably tighten this up slightly, though my suggestion might not be ideal for highlighting the backslash in the YouTube url??

```suggestion
It's important to note that the include and exclude rules will always have regular expressions in effect. For rules to match, you may have to escape special characters that commonly appear in urls like `?`, `+`, or `.`. by placing a `\` before the character. For example: `youtube.com/watch\?rdwz7QiG0lk`
```

Reply:
I think you can still highlight it by putting it in bold? I am not sure if it was highlighted originally, but if it was I probably did it by accident :]

It's a nice addition to highlight it, but it's not necessary and I think having a consistent single quote throughout every example is a better compromise.

> commonly appear in urls like `?`, `+`, or `.`**.** by placing a `\` before the character

I think the dot after the list of special characters to escape should be removed. I don't know how to edit your suggestion. 

I appreciate the suggestions, it's more concise and to the point with better pacing.